cli/configssh_test.go

```diff
@@ -611,6 +611,18 @@ func TestConfigSSH_FileWriteAndOptionsFlow(t *testing.T) {
 				regexMatch: "RemoteForward 2222 192.168.11.1:2222.*\n.*RemoteForward 2223 192.168.11.1:2223",
 			},
 		},
+		{
+			name: "Hostname Suffix",
+			args: []string{
+				"--yes",
+				"--hostname-suffix", "testy",
+			},
+			wantErr:  false,
+			hasAgent: true,
+			wantConfig: wantConfig{
+				regexMatch: `ProxyCommand .* ssh .* --hostname-suffix testy %h`,
```

Comment:
Nit: Would be nice to test the host header here as well (i.e. that `Host *.testy` is  present in SSH config).